app/api/ltc.py

```diff
@@ -126,6 +126,33 @@ def collapse_rows_new_header_names(df_group, col_map):
     return new_df_subset
 
 
+# cleans up CTP Facility Types and federal/state regulated
+# this is optimized for FL - other states have different labels
+def state_to_ctp(record):
+    state = record['State_Facility_Type']
+    if(state == 'ALF' or state == 'Assisted Living'):
+        record['CTP_Facility_Type'] = 'Assisted Living'
+        record['Regulate'] = 'State'
+    elif(state == 'NH'):
+        record['CTP_Facility_Type'] = 'Nursing Home'
+        record['Regulate'] = 'Federal'
+    elif(state == 'ICF'):
+        record['CTP_Facility_Type'] = 'Other'
+        record['Regulate'] = 'State'
+    else:
+        record['CTP_Facility_Type'] = None
+        record['Regulate'] = None
+    return record
+
+
+# clears any CMS IDs tied to facilities that are not nursing homes
+# this is optimized for FL - other states have different labels
+def clear_non_nh_cms_ids(record):
+    if ((record['State_Facility_Type'] != 'NH') and ~(record['CMS_ID'] == record['CMS_ID'])):
```

Comment:
the 2nd half of that looks fishy - record['CMS_ID'] would always equal itself?

Reply:
I was using the fact that nan != nan (so if the record had a valid CMS ID this condition would be true) but I don't actually need that check at all. ty!